TrkDiag/src/BkgDiag_module.cc

```diff
@@ -360,21 +362,19 @@ namespace mu2e
   bool BkgDiag::findData(const art::Event& evt){
     _chcol = 0; _shfcol = 0; _bkgccol = 0; _bkgqcol = 0; _mcdigis = 0;
```

Comment:
To be honest, I'm not fond of having a million pointers for the data products, it makes the class bloated. But you get a free pass for this time

Reply:
This is old code, it works, I prefer to leave it.